### PR TITLE
fix: add examples fashion

### PR DESCRIPTION
**Pull request by davidbp**
This PR changes the formatting of fashion data so that encoders from jinahub can be used directly with a  shared suitable input format.

### CODE REVIEW

File: .github/pages/hello-world.md [hunk 1]
```diff
@@ -35,6 +35,33 @@ This downloads the Fashion-MNIST training and test dataset and tells Jina to ind
...
 
+### Changing the embedding method
+
+You can run the `jina hello fashion` demo using a different embedding method from [ImageTorchEncoder](https://github.com/jina-ai/executor-image-torch-encoder)
+simply changing in `jina/helloworld/fashion/app.py` the flow
+
+```python
+    f = (
+        Flow()
+        .add(uses=MyEncoder, parallel=1)
```
Comment (JoanFM) [inline .github/pages/hello-world.md:46]: parallel=1 should be fixed already in the next bump of jina

File: .github/pages/hello-world.md [hunk 2]
```diff
@@ -35,6 +35,33 @@ This downloads the Fashion-MNIST training and test dataset and tells Jina to ind
 Then it randomly samples images from the test set as queries and asks Jina to retrieve relevant results.
 The whole process takes about 1 minute.
 
+### Changing the embedding method
```
Comment (JoanFM) [inline .github/pages/hello-world.md:38]: I would name use `jinahub executors`

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: .github/pages/hello-world.md (hunk 2 above)
Comment (davidbp) [reply .github/pages/hello-world.md]: This will be changed in another PR

File: jina/helloworld/fashion/app.py [hunk 3]
```diff
@@ -73,7 +73,7 @@ def hello_world(args):
     # load index flow from a YAML file
     f = (
         Flow()
-        .add(uses=MyEncoder, parallel=2)
+        .add(uses=MyEncoder, parallel=1)
```
Comment (JoanFM) [inline jina/helloworld/fashion/app.py:76]: do not change this

File: jina/helloworld/fashion/helper.py [hunk 4]
```diff
@@ -66,11 +72,14 @@ def query_generator(num_docs: int, target: dict, with_groundtruth: bool = True):
     :param with_groundtruth: True if want to include labels into query data
     :yields: query data
     """
+    random.seed(seed)
```
Comment (JoanFM) [inline jina/helloworld/fashion/helper.py:75]: Remove this, the point is to have variability

File: jina/helloworld/fashion/my_executors.py [hunk 5]
```diff
@@ -59,13 +59,24 @@ def __init__(self, **kwargs):
...
         embeds = (content.reshape([-1, 784]) / 255) @ self.oth_mat
         for doc, embed in zip(docs, embeds):
             doc.embedding = embed
             doc.convert_image_blob_to_uri(width=28, height=28)
             doc.pop('blob')
 
 
+class Converter(Executor):
+    @requests
+    def encode(self, docs: 'DocumentArray', **kwargs):
```
Comment (JoanFM) [inline jina/helloworld/fashion/my_executors.py:74]: do not call it encode

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: jina/helloworld/fashion/app.py (hunk 3 above)
Comment (JoanFM) [reply jina/helloworld/fashion/app.py:76]: do not change `parallel`

**Review by JoanFM — DISMISSED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**